gradle.properties

```diff
@@ -40,7 +40,7 @@ android.nonTransitiveRClass=true
 # Disable default features used by every module
 # https://developer.android.com/studio/releases/gradle-plugin#buildFeatures
 android.defaults.buildfeatures.aidl=false
-android.defaults.buildfeatures.buildconfig=false
+android.defaults.buildfeatures.buildconfig=true
```

Comment:
This has negative impact on build perfs. Do we really need that?

Reply:
Oops, i forgot to restore it. 
Done